ddtrace/debugging/_encoding.py

```diff
@@ -206,11 +206,9 @@ def _qualname(_type):
             except AttributeError:
                 return repr(_type)
         try:
```

Comment:
this logic now seems to be same as if module == "__builtin__" - maybe this can be simplified?